src/shared/src/types/change-request-types.ts

```diff
@@ -29,7 +29,7 @@ export const ChangeRequestType = {
   Activation: 'ACTIVATION'
 } as const;
 // eslint-disable-next-line @typescript-eslint/no-redeclare
-export type ChangeRequestType = typeof ChangeRequestType[keyof typeof ChangeRequestType];
+export type ChangeRequestType = (typeof ChangeRequestType)[keyof typeof ChangeRequestType];
```

Comment:
Is there a reason for this change?

Reply:
If I type the formatting shortcut Shift + Alt + F, then it does that automatically